templates/patterns/card/pattern-card.html.twig

```diff
@@ -4,52 +4,51 @@
  * Card pattern.
  */
 #}
-{% spaceless %}
-{% if title is not empty %}
-  {% set title = {
-    'content': title,
-  } %}
+{% apply spaceless %}
+{% set content %}
+  {% if meta is not empty %}
+    {# Use negative margin to compensate for me-3 on each item. #}
+    <div class="my-3 me-n3">
+      {% for _meta_item in meta %}
+        <span class="text-muted me-3">{{ _meta_item }}</span>
+      {% endfor %}
+    </div>
+  {% endif %}
+  <div>{{ content }}</div>
```

Comment:
is the compulsory div intentional ?

Reply:
I think @brummbar  mentioned we should wrap this in a div. I don't have a strong opinion on it myself. 
In earlier versions of the theme, the idea was that "content" is like a general-purpose variable where developers can put additional stuff that is not already handled with other variables. so it would support more than one element, and these elements would already have their own wrappers.

This said: If we do keep the div, we need to check first that the content is not empty.

So we can either drop the div, or check emptiness first.